### ISSUE TITLE
Add school year to program records

### ISSUE THREAD

**Issue by ghing** [ghing/learning-collaborative-maps#73]
In order to organize program records in the school detail view and to have the map reflect the current year's programming, we'll need to add a school year field to the program records.

Geoff thinks the easiest way to format this is to just use the numeric year that is the start of the school year, e.g.

`school_year: 2017` 

for the 2017-2018 school year.

* [ ] Identify a program that can edit data in the MongoDB database 
* [ ] Add school year field to each program record

**Comment by ghing** [comment]
In playing around with database admin tools, I found that there were 141 schools with at least one program.  Many have multiple programs, so the number of records is likely to be higher.  @Mattwalsh129 is that a reasonable number of records to update by September?

**Comment by Mattwalsh129** [comment]
Yes I think I can make this happen. Is this something that is ready to be
done right now?

**Comment by ghing** [comment]
@Mattwalsh129 This is something that can be done right now.  We have to make sure the database management program can run on your computer however.  Can you tell me what version of MacOS X you're running?

**Comment by Mattwalsh129** [comment]
I am currently working from a dell computer from work. Its windows 7? I am
not sure what more information you may need

**Comment by ghing** [comment]
@Mattwalsh129 Ah, cool.  I imagined you'd be doing this on your personal laptop.  Windows 7 should work.  You should try to download and install the program at https://mongobooster.com/downloads and let me know if you're able to open it.  If so, I can give you more details about editing records, but that will probably have to happen in person.

**Comment by Mattwalsh129** [comment]
Downloaded!

**Comment by ghing** [comment]
@Mattwalsh129 Were you able to run the program, at least enough to see the initial screens.

**Comment by Mattwalsh129** [comment]
yep (i think?)

[image: Inline image 1]
